src/modules/notifications/NotificationsManager.js

```diff
@@ -0,0 +1,33 @@
+// @flow
+
+import messaging from '@react-native-firebase/messaging'
+
+const requestUserPermissionForPushNotifications = async () => {
+  const authStatus = await messaging().requestPermission()
+
+  const enabled =
+    authStatus === messaging.AuthorizationStatus.AUTHORIZED ||
+    authStatus === messaging.AuthorizationStatus.PROVISIONAL
+
+  if (enabled) {
+    console.debug('Authorization status:', authStatus)
+  }
+}
+
+const unSubscribeToPreviousCityTopic = async (prevCity, prevLanguage) => {
+  messaging()
+    .unsubscribeFromTopic(`${prevCity}-${prevLanguage}-news`)
+    .then(() => console.debug(`Unsubscribed from ${prevCity}-${prevLanguage}-news topic!`))
```

Comment:
Shouldn't we log if it fails? Same for `subscribeToNewsCityTopic`.

Reply:
yes we should, i will add it

Reply:
BTW I already added a try/catch on the places it is invoked, shouldn't this be enough?

Reply:
Yep, should be enough :+1: